docs/layouts/custom-slots.md

```
@@ -18,69 +19,13 @@
-console.log('Custom description slot')
-\`\`\`
-
-</template>
-
-</OAOperation>
+<!--@include: ./parts/custom-slots-example.md-->
 ```
 
 This will render the following:
 
 ::: info Custom description slot 
```

Markdownlint

Expected: 0 or 2; Actual: 1
Trailing spaces

(MD009, no-trailing-spaces)